corehq/apps/userreports/templates/userreports/summary_data_source.html

```diff
@@ -72,7 +72,7 @@ <h2>Named Filters</h2>
       <tr>
         <td id="{{ named_filter_prefix }}:{{ filter.name }}">{{ filter.name }}</td>
         <td>{{ filter.comment }}</td>
-        <td><pre>{{ filter.readable_output|safe }}</pre></td>
+        <td><pre>{{ filter.readable_output }}</pre></td>
```

Comment:
Can't you also remove `safe` in the `expression.readable_output|safe` and `indicator.readable_output|safe` expressions further up in this file, since you updated the `readable_filter` returned by `named_expression_summary` and `indicator_summary`?

Reply:
Thanks for catching this